site/tutorials/tutorial-two-java.md

```diff
@@ -463,7 +464,7 @@ And our `Worker.java`:
             }
           }
         };
-        channel.basicConsume(TASK_QUEUE_NAME, false, consumer);
+        channel.basicConsume(TASK_QUEUE_NAME, consumer);
```

Comment:
Ah, OK. No need to edit anything, I need to read the whole thing before making any conclusions, sorry.